Build failure with GCC 15.1.1 in lib_battery_dispatch.cc

lib_battery_dispatch.cc is missing a `<cstdint>` include for the `SIZE_MAX` symbol. 

```
/home/josiah/code/open_contrib/ssc/shared/lib_battery_dispatch.cpp: In constructor ‘dispatch_automatic_t::dispatch_automatic_t(battery_t*, double, double, double, int, double, double, double, double, double, double, double, int, int, int, std::size_t, std::size_t, double, bool, bool, bool, bool, bool, std::vector<double>, int, std::vector<double>, std::vector<double>, double, bool, bool, bool, double)’:
/home/josiah/code/open_contrib/ssc/shared/lib_battery_dispatch.cpp:680:26: error: ‘SIZE_MAX’ was not declared in this scope
  680 |     _hour_last_updated = SIZE_MAX;
      |                          ^~~~~~~~
/home/josiah/code/open_contrib/ssc/shared/lib_battery_dispatch.cpp:39:1: note: ‘SIZE_MAX’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   38 | #include <math.h>
  +++ |+#include <cstdint>
   39 | #include <algorithm>
```

**Desktop (please complete the following information):**
 - OS: OpenSUSE Tumbleweed

--- a/shared/lib_battery_dispatch.cpp
+++ b/shared/lib_battery_dispatch.cpp
@@ -37,6 +37,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <math.h>
 #include <algorithm>
+#include <cstdint>
 
 /*
 Dispatch base class

